FHEM/00_SIGNALduino.pm

```diff
@@ -1421,14 +1425,14 @@ my %ProtocolListSIGNALduino  = (
 			preamble	=> '810c04xx0909a001',
 			length_min	=> '59',
 			length_max	=> '67',
-			#postDemodulation => &SIGNALduino_postDemo_FHT80,
+			postDemodulation => \&SIGNALduino_postDemo_FHT80,
 		},
 	"74" => ## FS20 - 'Remote Control (868Mhz),  @HomeAutoUser
 		{
 			name		=> 'FS20',
 			comment		=> 'reserviert, Remote Control (868Mhz)',
 			id		=> '74',
-			developId	=> 'p',
+			#developId	=> 'p',
```

Comment:
ich würde bei FHT80TF, FHT80 und FS20 erstmal
`developId    => 'y',`
eintragen, das ganze sieht recht komplex aus, oder habt ihr das ganze vollständig durchgetestet?

Reply:
MC zu deaktivieren ist natürlich nur ein Workaround und keine Lösung.

Reply:
Welchen Sinn macht es bei 868Mhz MC zu aktivieren, wenn keine 868Mhz Sensoren mit manchester Protokolle bekannt sind?

Reply:
Ich lasse die FHT80TF, FHT80 und FS20 seit ca. 2Wochen laufen. Es läuft, Problem ist nur immer das Problem mit dem Verschieben im Puffer aufgrund der Signallänge, wenn angeblich die "MC/MU" Erkennung funktionieren sollte.

"funktioniert es zuverlässig, wenn Du das MC deaktivierst?"

MC ausgeschalten und es läuft. Zuverlässig läuft der Code. Wenn ein Signal richtig "durchkommt" alles in Ordnung. Wenn das Signal natürlich aufgrund des "Pufferproblems" nicht richtig verarbeitet wird, so sinkt natürlich die erfolgreiche MSG Empfangsverarbeitung.